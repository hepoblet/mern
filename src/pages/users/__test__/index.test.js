import React from 'react';
import { render } from '@testing-library/react';
import {Â create } from 'react-test-renderer';
import UsersList from '../index';

describe('UsersList tests', () => {
  test('render users list correctly', () => {
    render(<UsersList />);
  });

  test('users list snapshot', () => {
    const list = create(<UsersList />);
    expect(list.toJSON()).toMatchSnapshot();
  });
});
