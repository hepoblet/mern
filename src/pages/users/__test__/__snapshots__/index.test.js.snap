// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList tests users list snapshot 1`] = `
<div>
  <h3>
    Users List
  </h3>
  <table
    className="table"
  >
    <thead
      className="thead-dark"
    >
      <tr>
        <th>
          Username
        </th>
        <th>
          Actions
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
